src/bitcoin-wallet.cpp

```diff
@@ -28,15 +30,18 @@ static void SetupWalletToolArgs()
     gArgs.AddArg("-wallet=<wallet-name>", "Specify wallet name", ArgsManager::ALLOW_ANY | ArgsManager::NETWORK_ONLY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-debug=<category>", "Output debugging information (default: 0).", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     gArgs.AddArg("-printtoconsole", "Send trace/debug info to console (default: 1 when no -debug is true, 0 otherwise).", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
+    if (include_ipc) {
+        gArgs.AddArg("-ipcconnect=<address>, -noipcconnect", "Connect to bitcoin-node process in the background to perform online operations. Valid <address> values are 'auto' to try connecting to default socket in <datadir>/sockets/node.sock, but proceed offline if it isn't available, 'unix' to connect to the default socket and fail if it isn't available, 'unix:<socket path>' to connect to a socket at a nonstandard path. Default value: auto", ArgsManager::ALLOW_ANY, OptionsCategory::IPC);
```

Comment:
I think this is the only case of multiple options separated by `,` inside the first argument of `ArgsManager::AddArg`. I don't know if it's strictly wrong, as it's not currently parsed in any way, but maybe the idea is to do so in the future. @MarcoFalke?

Reply:
No plans right now to touch the command line help in any way, so this might be fine as is